packages/deployment/app/config.yml

```diff
@@ -111,6 +111,8 @@ default:
             function: Fn::Sub
             value: '${AWS::Region}'
     docker:
+      # Expected auth types are 'default' and 'iam'
+      authType: default
```

Comment:
Is there a more descriptive authType than default?

Reply:
Great question. One option is `docker`, as that's the `ENGINE_AUTH_TYPE` that is actually being set, but I think that is maybe too verbose. With the way things are currently written, `dockerhub` and `ecr` would be just as valid flags.

Reply:
With `dockerhub` and `ecr` there would be little confusion as to what each does, I think. In that case I think `authType` would still work as a label... I'm having difficulties thinking of something more appropriate.

Reply:
maybe `repoHost: dockerhub | ecr` would be better